packages/mobile/src/in-house-liquidity/index.test.ts

```diff
@@ -182,4 +184,63 @@ describe('In House Liquidity Calls', () => {
       expect(response).toBeInstanceOf(Response)
     })
   })
+  describe('createFinclusiveBankAccount', () => {
+    it('calls the /account/bank-account endpoint', async () => {
+      mockFetch.mockResponseOnce(JSON.stringify({}), { status: 201 })
+      const response = await createFinclusiveBankAccount({
+        accountMTWAddress: MOCK_USER.accountMTWAddress,
+        walletAddress: MOCK_USER.walletAddress,
+        plaidAccessToken: 'foo',
+      })
+      const expectedBody = JSON.stringify({
+        accountAddress: MOCK_USER.accountMTWAddress,
+        plaidAccessToken: 'foo',
+      })
+
+      // Calls Fetch Correctly
+      expect(mockFetch).toHaveBeenCalledWith(
+        `${networkConfig.inHouseLiquidityURL}/account/bank-account`,
+        {
+          body: expectedBody,
+          headers: {
+            Date: expect.anything(),
+            Authorization: expect.stringContaining(`Valora ${MOCK_USER.walletAddress}:`),
```

Comment:
you could merge the ihl auth branch into this to start working of it if you wanted. it's stable at this point

Reply:
to start fixing up this auth stuff